<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewReferralForm</class>
 <widget class="QWidget" name="NewReferralForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>316</width>
    <height>146</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Новое направление</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Регистрационный №:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox_regID"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_">
     <property name="text">
      <string>ФИО врача:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Дата направления:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDateEdit" name="dateEdit">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Время направления:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QTimeEdit" name="timeEdit">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QPushButton" name="pushButton_ok">
     <property name="text">
      <string>Продолжить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
